src/components/RecentBlogPosts/index.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import posts from '@site/.docusaurus/docusaurus-plugin-content-blog/default/blog-posts-metadata.json';
+import RecentBlog from '../RecentBlogPost';
```

Comment:
```suggestion
import RecentBlogPost from '../RecentBlogPost';
```

Reply:
Perhaps, `RecentBlogPost` can be included directly into `RecentBlogPosts/index.jsx`; up to you.